core/arch/arm/plat-marvell/kern.ld.S

```diff
@@ -0,0 +1 @@
+#include "../kernel/kern.ld.S"
```

Comment:
minor: you can remove this file (since https://github.com/OP-TEE/optee_os/pull/1873)